concent_api/core/views.py

```diff
@@ -53,7 +50,7 @@ def send(_request: HttpRequest, client_message: Message, client_public_key: byte
 @handle_errors_and_responses(database_name='control')
 def receive(_request: HttpRequest, message: Message, _client_public_key: bytes) -> Union[Message, HttpResponse]:
     assert isinstance(message.client_public_key, bytes)
-    update_timed_out_subtasks(
+    update_all_clients_timed_out_subtasks(
```

Comment:
Instead of splitting this into two functions, you could add `client_message` as an optional parameter to original `update_timed_out_subtasks` and limit the queryset depending on its presence.